website_cms/templates/misc.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <template id="website.publish_management">
+
+    <!-- we do not want additional dropdown for cms.page -->
+    <!--
+    this was supposed to work (tested w/ chrome) but looks like odoo does not accept it :S rewrite everything...
+      <xpath expr="//button[contains(@class, 'dropdown-toggle')]/following-sibling::ul" position="replace" />
```

Comment:
You should always try to change the least possible, but if that's not possible, then override it in a separate view. But never write a view outside your model's scope. This template's id is `website.*`, that's not acceptable because of what I said just above.